packages/x-data-grid/src/components/columnsManagement/GridColumnsManagement.tsx

```diff
@@ -255,7 +255,7 @@ function GridColumnsManagement(props: GridColumnsManagementProps) {
           </GridColumnsManagementEmptyText>
         )}
       </GridColumnsManagementBody>
-      {!disableShowHideToggle && !disableResetButton && currentColumns.length > 0 ? (
+      {currentColumns.length > 0 ? (
```

Comment:
Thanks!
Nitpick: we only want to render the footer if at least one of the buttons is not disabled:

```suggestion
      {(!disableShowHideToggle || !disableResetButton) && currentColumns.length > 0 ? (
```